libs/cardano-ledger-core/src/Cardano/Ledger/Core/Era.hs

```diff
@@ -207,8 +208,14 @@ fromEraCBOR :: forall era t s. (Era era, DecCBOR t) => Plain.Decoder s t
 fromEraCBOR = eraDecoder @era decCBOR
 {-# INLINE fromEraCBOR #-}
 
+-- | Convert a type that implements `DecCBOR` to plain `Plain.Decoder` using the lowest
```

Comment:
Seems out of sync with the signature, where the constraint is a type that implements `DecShareCBOR`, unless the distinction is not relevant here

Reply:
Nevermind, I just saw it's fixed in a later commit!